esmvalcore/preprocessor/_ancillary_vars.py

```diff
@@ -46,6 +49,17 @@ def _load_fx(fx_info, check_level):
     return fx_cube
 
 
+def _is_fx_broadcastable(fx_cube, cube):
+    try:
+        da.broadcast_to(fx_cube.core_data(), cube.shape)
+    except ValueError as exc:
+        logger.debug("Dimensions of %s and %s cubes do not match. "
+                     "Discarding use of fx_variable: %s",
+                     cube.var_name, fx_cube.var_name, exc)
```

Comment:
This would print the exception message from dask:

`2021-06-01 10:15:51,152 UTC [17894] DEBUG   Dimensions of o2 and volcello cubes do not match. Discarding use of fx_variable: cannot broadcast shape (12, 75, 292, 362) to shape (1, 75, 292, 362)`

Reply:
nice, thanks Saskia! it does relay good information indeed.